tests/phpunit/tests/link/getAdjacentPost.php

```diff
@@ -350,4 +350,81 @@ public function filter_excluded_terms( $excluded_terms ) {
 		$excluded_terms[] = $this->exclude_term;
 		return $excluded_terms;
 	}
+
+	/**
+	 * @ticket 41131
+	 */
+	public function test_get_adjacent_post_cache() {
+		global $wpdb;
+		// Need some sample posts to test adjacency
```

Comment:
```suggestion
		// Need some sample posts to test adjacency.
```

Use a period at the end of each inline document: https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-1-single-line-comments